bin/apisix

```diff
@@ -86,7 +86,9 @@ local template = require("resty.template")
 local ngx_tpl = [=[
 # Configuration File - Nginx Server Configs
 # This is a read-only file, do not try to modify it.
-
+{% if enable_root then %}
+user root;
```

Comment:
I do not think that is a good idea.

Reply:
Agree, we should encourage running APISIX with least privilleges not otherwise.